integreat_cms/cms/models/regions/region.py

```diff
@@ -418,9 +419,7 @@ class Region(AbstractBaseModel):
     zammad_url = models.URLField(
         max_length=256,
         blank=True,
-        null=True,
```

Comment:
`IntegrityError` occurs when a region is created without Zammad-URL 👀

Reply:
wild ... thanks :)

Reply:
Yeah okay, we had our self defined clean method that returned `None`.